.env.client-public

```diff
@@ -0,0 +1,26 @@
+# General
+NODE_ENV=development
+BRFENERGI_LANG=sv
+PORT=3000
+
+# Services
+MAPBOX_ACCESS_TOKEN=<MAPBOX_TOKEN>
+MAPBOX_STYLE=mapbox://styles/tornqvist/ciye3wh1c000s2sqyhepqpnrz
+GOOGLE_ANALYTICS_ID=<ANALYTICS_ID>
+
+# OAuth
+BRFENERGI_SERVICE_URL=http://localhost:3000
+BRFENERGI_SESSION_SECRET=<SESSION_SECRET>
```

Comment:
`BRFENERGI_SESSION_SECRET ` is only used on the server for signing cookies and doesn't need to be included in the client.